src/screens/FeedScreen/ContentLink.js

```diff
@@ -11,7 +11,7 @@ import {useNavigation, useRoute} from '@react-navigation/native';
 import Card from '../../components/Card/Card';
 import TopicsChip from '../../components/TopicsChip/TopicsChip';
 import {COLORS} from '../../utils/theme';
-import {fonts, normalizeFontSize, normalizeFontSizeByWidth} from '../../utils/fonts';
+import {fonts, normalize, normalizeFontSize, normalizeFontSizeByWidth} from '../../utils/fonts';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement has been updated to include the `normalize` function from `../../utils/fonts`. This is a good change as it seems like this function will be used for height calculations in this component. However, please ensure that the `normalize` function is properly defined and exported in the `fonts.js` file.
```diff
- import {fonts, normalizeFontSize, normalizeFontSizeByWidth} from '../../utils/fonts';
+ import {fonts, normalize, normalizeFontSize, normalizeFontSizeByWidth} from '../../utils/fonts';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->